sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -397,6 +420,29 @@ def test_y_cable_wrapper_get_presence_with_platform_chassis(self, mock_chassis):
 
         assert(presence == True)
 
+
+    @patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_platform_chassis')
+    def test_y_cable_wrapper_get_presence_with_platform_chassis_raise_exception(self, mock_chassis):
+
+        mock_object = MagicMock(spec=SfpBase)
+        mock_object.get_presence = MagicMock(side_effect = NotImplementedError)
+        mock_chassis.get_sfp = MagicMock(return_value = mock_object)
+
+        class mock_sfp_util:
+            def get_presence():
+                return True
+
+        y_cable_platform_sfputil = mock_sfp_util()
+
+        #ASHWIN: TODO: How can I override the local y_cable_platform_sfputil 'NoneType' class object with my mock object containing get_presence method? 
```

Comment:
remove name, do this throughout the file

Reply:
Same as above.

Reply:
Done in latest commit.